packages/samples/vuejs/README.md

```diff
@@ -1,46 +1,157 @@
-# vue-search
+# Vue.js Search Page
 
-This template should help get you started developing with Vue 3 in Vite.
+## Project setup
 
-## Recommended IDE Setup
+```sh
+npm install
+```
+
+### Compiles and hot-reloads for development
+
+```sh
+npm run serve
+```
+
+### Compiles and minifies for production
+
+```sh
+npm run build
+```
 
-[VSCode](https://code.visualstudio.com/) + [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar) (and disable Vetur) + [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.vscode-typescript-vue-plugin).
+### Customize configuration
+
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
+&nbsp;
+
+## Configuring `@coveo/atomic`
+&nbsp;
+
+### Add Vue configuration to help it handle atomic's components
+
+```js
+// vue.config.js
+{
+  ...,
+  chainWebpack: (config) => {
+    // lets us parse html templates, which will be necessary for atomic components that require the native <template> tag
+    config.module
+      .rule('html')
+      .test(/\.html$/)
+      .use('html-loader')
+      .loader('html-loader');
+
+    // tells Vue to treat any components starting with 'atomic-' as native components, and therefore not try and create Vue components out of them
+    config.module
+      .rule('vue')
+      .use('vue-loader')
+      .tap((options) => ({
+        ...options,
+        compilerOptions: {
+          isCustomElement: (tag) => tag.startsWith('atomic-'),
+        },
+      }));
+  }
+}
+```
 
-## Type Support for `.vue` Imports in TS
+### Add dependencies
 
-TypeScript cannot handle type information for `.vue` imports by default, so we replace the `tsc` CLI with `vue-tsc` for type checking. In editors, we need [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.vscode-typescript-vue-plugin) to make the TypeScript language service aware of `.vue` types.
+`html-loader` is necessary for the configuration above to work and
+`ncp` is a utility that we will use to copy some assets from the atomic project into the vue project in the next step
```

Comment:
I think we should explain why it's needed to copy assets in the public folder, and what happens if this is not done/what symptom will appear.

ie: Labels will appear as temporary placeholder.

Just so that someone that forget to do this part, then comes back to the doc to troubleshoot will easily spot  the fix/resolution.